### PR TITLE
Network acl rules

### PR DESCRIPTION
**Pull request by franzpgarcia**

### CODE REVIEW

File: cloudca/README.md [hunk 1]
```diff
@@ -113,6 +114,42 @@ The following arguments are supported:
...
+- rule_number - (Required) Rule number of the network ACL rule
+- cidr - (Required) Cidr of the network ACL rule
+- action - (Required) Action of the network ACL rule (i.e. Allow or Deny)
+- protocol - (Required) Protocol of the network ACL rule (i.e. TCP, UDP, ICMP or All)
+- traffic_type - (Required) TrafficType of the network ACL rule (i.e. Ingress or Egress)
+- icmp_type - (Optional) The ICMP type. Can only be used with ICMP protocol
+- icmp_code - (Optional) The ICMP code. Can only be used with ICMP protocol
+- start_port - (Optional) The start port. Can only be used with TCP/UDP protocol
+- end_port - (Optional) The end port. Can only be used with TCP/UDP protocol
+- network_acl_id - (Required) ID of the network ACL where the rule should be created
```
Comment (simongodard) [inline cloudca/README.md]: We should move this argument below environment_name since it's required.

Comment (franzpgarcia) [inline cloudca/README.md]: Done

**Comment by pdube** [comment]
👍

**Comment by simongodard** [comment]
💯

File: cloudca/README.md [hunk 2]
```diff
@@ -113,6 +114,42 @@ The following arguments are supported:
...
+```
+resource "cloudca_network_acl_rule" "my_acl" {
+	service_code = "compute-east"
+	environment_name = "dev"
+	rule_number = 55
+	action = "Allow"
+	protocol = "TCP"
+	start_port = 80
+	end_port = 80
+	traffic_type = "Ingress"
```
Comment (prollynomial) [inline cloudca/README.md]: Are these case-sensitive? (ie traffic_type, action, protocol)

Comment (simongodard) [inline cloudca/README.md]: @prollynomial no, they are **case-insensitive**

**Comment by prollynomial** [comment]
LGTM

**Pull request merged**